tst/testinstall/meataxe.tst

```diff
@@ -106,47 +106,47 @@ true
 #
 gap> M4:=InducedGModule(SymmetricGroup(6),G,M);
 rec( IsOverFiniteField := true, dimension := 30, field := GF(7^2), 
-  generators := [ < immutable compressed matrix 30x30 over GF(49) >, 
-      < immutable compressed matrix 30x30 over GF(49) > ], 
+  generators := [ < immutable compressed matrix 30x30 over GF(7^2) >, 
+      < immutable compressed matrix 30x30 over GF(7^2) > ], 
   isMTXModule := true )
 gap> SortedList(List(MTX.CompositionFactors(M4), m -> m.dimension));
 [ 1, 5, 5, 9, 10 ]
 
 #
 gap> M5:=WedgeGModule(M);
 rec( IsOverFiniteField := true, dimension := 10, field := GF(7^2), 
-  generators := [ < immutable compressed matrix 10x10 over GF(49) >, 
-      < immutable compressed matrix 10x10 over GF(49) > ], 
+  generators := [ < immutable compressed matrix 10x10 over GF(7^2) >, 
+      < immutable compressed matrix 10x10 over GF(7^2) > ], 
   isMTXModule := true )
 gap> SortedList(List(MTX.CompositionFactors(M5), m -> m.dimension));
 [ 4, 6 ]
 gap> cf:=MTX.CollectedFactors(M5);;
 gap> MTX.Distinguish(cf,1);
 gap> MTX.Distinguish(cf,2);
 gap> MTX.BasesSubmodules(M5);
-[ [  ], < immutable compressed matrix 4x10 over GF(49) >, 
-  < immutable compressed matrix 6x10 over GF(49) >, 
-  < immutable compressed matrix 10x10 over GF(49) > ]
+[ [  ], < immutable compressed matrix 4x10 over GF(7^2) >, 
+  < immutable compressed matrix 6x10 over GF(7^2) >, 
+  < immutable compressed matrix 10x10 over GF(7^2) > ]
 gap> MTX.BasesMinimalSubmodules(M5);
-[ < immutable compressed matrix 4x10 over GF(49) >, 
-  < immutable compressed matrix 6x10 over GF(49) > ]
+[ < immutable compressed matrix 4x10 over GF(7^2) >, 
+  < immutable compressed matrix 6x10 over GF(7^2) > ]
 gap> MTX.BasesMaximalSubmodules(M5);
-[ < immutable compressed matrix 6x10 over GF(49) >, 
-  < immutable compressed matrix 4x10 over GF(49) > ]
+[ < immutable compressed matrix 6x10 over GF(7^2) >, 
+  < immutable compressed matrix 4x10 over GF(7^2) > ]
 gap> MTX.BasisRadical(M5);
 [  ]
 gap> MTX.BasisSocle(M5);
-< immutable compressed matrix 10x10 over GF(49) >
+< immutable compressed matrix 10x10 over GF(7^2) >
 gap> subs:=SMTX.MinimalSubGModules(M2,M5);
-[ < immutable compressed matrix 4x10 over GF(49) > ]
+[ < immutable compressed matrix 4x10 over GF(7^2) > ]
```

Comment:
I have not yet had a chance to look at the rest of this PR (it is rather big; the bigger the PR, the longer it takes until someone finds time to review it properly).

But I can already say that this change is problematic, because it would break the test suites and/or manual examples of a bunch of GAP packages. Is it really necessary to make this change to the printing here? If we really want to do that, it would be better to have it in its own, isolated PR, where we then can discuss plans for a migration. Affected packages include:
- atlasrep
- classicpres
- ctbllib
- fining
- gbnp
- orb
- semigroups
- spinsym

Reply:
The change isn't necessary, and is easy to reverse, I'm happy to do it. I'm also happy to split this PR up in to smaller chunks if that's desirable.